eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": [
+        "standard"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12
+    },
+    "rules": {
+        "no-var": "off"
+    }
```

Comment:
Do we really need anything other than the `"extends"` and `"rules"` sections?

Reply:
Didn't know if you want extra funct. I will remove

Reply:
@jsumners ESLint needs parserOptions to know which one should use - **ESLint allows you to specify the JavaScript language options you want to support. By default, ESLint expects ECMAScript 5 syntax** should I leave  "ecmaVersion": 12 ?. Do we need any "global variables that are predefined" https://eslint.org/docs/4.0.0/user-guide/configuring#specifying-environments  or you want me to remove them?

Reply:
@mcollina what is your opinion of this configuration? It eliminated needing to scatter inline tool directives throughout the source code, but it is more scaffolding than just "install `standard`". Unfortunately, `standard` explicitly ignores extra configuration by design. So we can't disable this rule while only using the `standard` package.

Reply:
I don't really understand why we are moving away from standard. I think it was ok.

Reply:
We still use the standard rule set, but gain the ability to disable its `var` rule in one place instead of having to do it on every line where we want to keep using `var`.

Reply:
Ok!

Reply:
Alrighty. @salesh let's rename the rc back to what it was and update the npm script. Then I'm good with this PR.